README.md

```diff
@@ -45,6 +45,9 @@ Note to install the software below we recommend the following package managers:
   - Create an empty database and accompanying user for use by the app (E.g. development database *elmo_d* with username *elmo*)
   - Set up mysql for timezone support: See [doc here](http://dev.mysql.com/doc/refman/5.5/en/time-zone-support.html)
 
+1. **GraphViz**
+  - Install GraphViz from http://graphviz.org/Download..php
```

Comment:
Can you make this one look like the others (minimum version, short description of what it's for, put it near the bottom and put 'development only' since it's not needed for production, remove the download link since folks may want to use their package manager (it's fine to put a link explaining what the software is though).